src/actions/promote.ts

```diff
@@ -77,22 +80,27 @@ async function promoteCanary(kubectl: Kubectl, manifests: string[]) {
       core.startGroup(
          'Deploying input manifests with SMI canary strategy from promote'
       )
-      await SMICanaryDeploymentHelper.deploySMICanary(
+
+      promoteResult = await SMICanaryDeploymentHelper.deploySMICanary(
          manifestFilesForDeployment,
          kubectl,
          true
       )
+
       core.endGroup()
 
       core.startGroup('Redirecting traffic to stable deployment')
-      await SMICanaryDeploymentHelper.redirectTrafficToStableDeployment(
-         kubectl,
-         manifests
-      )
+      const stableRedirectManifests =
+         await SMICanaryDeploymentHelper.redirectTrafficToStableDeployment(
+            kubectl,
+            manifests
+         )
+      promoteResult.manifestFiles.push(...stableRedirectManifests)
```

Comment:
feels weird that we are pushing to a property of promote result.

Can we rename promote result to what they are used for later?

filesToAnnotate maybe? and instead of being type DeployResult it could just be a list of files. That would help with readability

Reply:
yeah agreed, doesn't make sense to push to the result/to bundle it that way for no reason. will update